jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/Resource.java

```diff
@@ -1233,4 +1234,82 @@ public interface Mount extends Closeable
     }
+
+    /**
+     * <p>Make a Resource containing a collection of other resources</p>
+     * @param resources multiple resources to combine as a single resource. Typically, they are directories.
+     * @return A Resource of multiple resources.
+     * @see ResourceCollection
+     */
+    public static ResourceCollection of(List<Resource> resources)
+    {
+        if (resources == null)
+            throw new IllegalArgumentException("No resources");
+
+        return new ResourceCollection(resources);
+    }
+
+    /**
+     * <p>Make a Resource containing a collection of other resources</p>
+     * @param resources multiple resources to combine as a single resource. Typically, they are directories.
+     * @return A Resource of multiple resources.
+     * @see ResourceCollection
+     */
+    public static ResourceCollection of(Resource... resources)
+    {
+        if (resources == null)
+            throw new IllegalArgumentException("No resources");
+
+        return new ResourceCollection(Arrays.asList(resources));
+    }
+
+    /**
+     * <p>Make a Resource containing a collection of other resources</p>
+     * @param resourceAsStrings multiple resources to combine as a single resource. Typically, they are directories.
+     * @return A Resource of multiple resources or null if no resources passed.
+     * @see ResourceCollection
+     * @throws IllegalArgumentException if a string cannot be converted to resource
+     */
+    public static ResourceCollection of(String... resourceAsStrings)
+    {
+        if (resourceAsStrings == null || resourceAsStrings.length == 0)
+            throw new IllegalArgumentException("No resources");
+
+        List<Resource> resources = new ArrayList<>();
+        for (String r : resourceAsStrings)
+        {
+            if (r == null)
+                continue;
+            r = r.trim();
+            if (StringUtil.isBlank(r))
+                continue;
+            try
+            {
+                resources.add(Resource.newResource(r));
+            }
+            catch (IOException e)
+            {
+                throw new IllegalArgumentException(e);
+            }
+        }
+        if (resources.size() == 0)
+            throw new IllegalArgumentException("No resources");
+
+        return new ResourceCollection(resources);
+    }
+
+    /**
+     * <p>Make a Resource containing a collection of other resources</p>
+     * @param csvResources multiple resources to combine as a single resource. Typically, they are directories.
+     * @return A Resource of multiple resources or null if no resources passed.
+     * @see ResourceCollection
+     * @throws IllegalArgumentException if a string cannot be converted to resource
+     */
+    public static ResourceCollection of(String csvResources)
+    {
+        if (StringUtil.isBlank(csvResources))
+            throw new IllegalArgumentException("No resources");
+
+        return of(StringUtil.csvSplit(csvResources));
```

Comment:
This doesn't support the old behavior of allowing `,` or `|` as delimiters in the cases where the filename has a comma of its own.

This is what fromList does.

Reply:
I eliminated fromList in the original ResourceCollection PR as it had a chicken/egg issue.
It's been replaced with `List<URI> Resource.split(String)` in the original ResourceCollection PR.